src/dbus/server.rs

```diff
@@ -93,6 +94,9 @@ impl Handler {
 
         let hints = Hints::from(&hints);
 
+        let summary = Text::parse(summary);
```

Comment:
Does the summary have HTML tags? Or did you leave them in just in case?

Reply:
it doesnt, i fixed it